stories/setup/Provider.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+// Without core/polyfill we see the following error:
+// "regeneratorRuntime is not defined".
+import 'core/polyfill';
```

Comment:
When I remove this, I don't see any errors. What are the steps to reproduce?

In any case, it's already imported in the webpack config so I'm not sure why it's needed twice. https://github.com/mozilla/addons-frontend/blob/5ad0920f1855cb28f51c6cf426791cd984b43f39/stories/setup/webpack.config.babel.js#L1

Reply:
actually I had to add this back with the changes I made. I removed this from the webpack tho which I didn't see had any impact.